src/openforms/scss/components/admin/_select.scss

```diff
@@ -1,3 +1,4 @@
 .form-variable-dropdown {
+  min-inline-size: var(--of-admin-select-min-inline-size);
   max-inline-size: var(--of-admin-select-max-inline-size);
```

Comment:
we could use the modern `clamp` property here: `inline-size: clamp($min, 100%, $max);` perhaps?